serverless/serverless-release-notes.adoc

```diff
@@ -22,10 +22,15 @@ include::modules/serverless-tech-preview-features.adoc[leveloffset=+1]
 include::modules/serverless-deprecated-removed-features.adoc[leveloffset=+1]
 
 // Release notes included, most to least recent
+// OCP + OSD + ROSA
+// include::modules/serverless-rn-1-23-0.adoc[leveloffset=+1]
```

Comment:
I'd leave this off for now, I can fix it in the PR for 1.23.0 release notes: https://github.com/openshift/openshift-docs/pull/45935